batch7crud-roll6/src/main/java/com/lftechnology/batch7crud/dao/UserDAOImpl.java

```diff
@@ -0,0 +1,137 @@
+package com.lftechnology.batch7crud.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.batch7crud.db.DbConnector;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.User;
+import com.lftechnology.batch7crud.util.TypeCaster;
+
+public class UserDAOImpl implements UserDAO {
+
+    private static final Logger logger = Logger.getLogger("Controller");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)